hips/draw/paint.py

```diff
@@ -36,6 +36,8 @@ class HipsPainter:
         Use the precise drawing algorithm
     progress_bar : bool
         Show a progress bar for tile fetching and drawing
+    use_aiohttp : bool
```

Comment:
I think a multiple-choice string option would be better, for two reasons:
1. it's extensible, in case we want to add a third implementation
2. it's clearer; with `use_aiohttp=False` and your parameter description, the user has no idea what they get.